tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -399,6 +439,30 @@ <h3>No profile data was found.</h3>
     detached: function() {
       this.set('_isAttached', false);
     },
+    openCaptureProfileDialog: function() {
+      this.$.captureProfileDialog.open();
+    },
+    shouldDisableCaptureProfileButton: function(profilerServiceAddress, profileDuration) {
```

Comment:
your changes in this file: we prefix private methods (since JS, not TS, does not have concept of private method, that's the naming convention we use). Would you be able to make that change? Thanks!